### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by TADraeseke**
Hi @markjdugger -- can you review and comment if you think this is clearer? Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,11 @@ ArcGIS Runtime API for Android v100.13.0 samples.  The `main` branch of this rep
 ## Developer Instructions
 Please read our [developer instructions wiki page](https://github.com/Esri/arcgis-runtime-samples-android/wiki/dev-instructions) to set up your developer environment with Android Studio.  Instructions include forking and cloning the repository for those new to Git.
 
+Once the project is cloned to disk you can import into Android Studio:
+
+* From the toolbar select **File > Open**, or **Open** from the Welcome Quick Start.
```
Comment (markjdugger) [inline README.md:14]:
```suggestion
* From the Welcome to Android Studio screen, click the **Open** button. (If you're already inside a project, click **File > Open** in the menu bar instead.)
```

File: README.md [hunk 2]
```diff
@@ -9,6 +9,11 @@ ArcGIS Runtime API for Android v100.13.0 samples.  The `main` branch of this rep
...
+* Navigate to **arcgis-runtime-samples-android/java** or **arcgis-runtime-samples-android/kotlin** project folder,  directory and click **OK**
```
Comment (markjdugger) [inline README.md:15]:
```suggestion
* Navigate to the **arcgis-runtime-samples-android/java** or **arcgis-runtime-samples-android/kotlin** folder and click **OK**.
```

**Review by markjdugger — CHANGES_REQUESTED** [review]

**Comment by markjdugger** [comment]
@TADraeseke I made edit suggestions above. I matched the wording with what we are using in the Android tutorials. (The Android tutorial changes are not merged yet, but you can see what we intend by examining the modified Java tutorials, already on the public website:
- https://developers.arcgis.com/java/maps-2d/tutorials/display-a-map/#create-a-new-java-project-with-gradle

IntelliJ IDEA and Android Studio, both from JetBrains, have very similar UI, so it makes sense to use the same wording for both Java and Android.

**Comment by TADraeseke** [comment]
Cool! I like you wording better. Made those changes.

**Pull request merged**